### PR TITLE
Define and activate aws 9.3.5 releases

### PR DESCRIPTION
**Pull request by sslavic**
Towards https://github.com/giantswarm/giantswarm/issues/12210

### CODE REVIEW

File: aws/v9.3.5/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# :zap: Giant Swarm Release v9.3.5 for AWS :zap:
+
+This release updates NGINX Ingress Controller to the latest upstream release.
+Most importantly update includes a fix for the regression introduced in the previous upstream release related to `use-regex` and `rewrite` annotations.
```
Comment (cokiengchiara) [inline aws/v9.3.5/README.md:4]:
```suggestion
Most importantly, it includes a fix for a regression introduced in the previous upstream release related to `use-regex` and `rewrite` annotations.
```

**Comment by tfussell** [comment]
Would you mind bumping Kubernetes to 1.16.12 to mitigate https://github.com/kubernetes/kubernetes/issues/92275 ?

**Comment by sslavic** [comment]
@tfussell I don't mind at all. Does it require new aws-operator release?

**Comment by tfussell** [comment]
> @tfussell I don't mind at all. Does it require new aws-operator release?

Thanks! No, only `containerlinux` should require a new operator. This is a minor change to the hyperkube image so it should only be necessary to change the `release.yaml` here. The description could be:
```
- Added a wrapper script for iptables in the recently upgraded hyperkube image to avoid crash looping of kube-proxy.
```

**Comment by sslavic** [comment]
@tfussell release definition has been updated, PTAL

**Comment by sslavic** [comment]
@MartaD with Chiara away this week, can you please help shipping these [few releases](https://gigantic.slack.com/archives/C90BZMUJ1/p1595232791397700) - review release notes, do the due diligence to communicate internal and with customer?

**Comment by tfussell** [comment]
> @tfussell release definition has been updated, PTAL

Looks good, thanks. FYI this is to fix the issue discovered with `kube-proxy` here https://github.com/giantswarm/giantswarm/issues/12252 and originally reported internally here https://gigantic.slack.com/archives/C52L7C6SC/p1592919849456500

**Comment by sslavic** [comment]
Tested on gauss:
- 9.3.5 cluster creation
- upgrade from 9.3.4
- upgrade from 9.0.7

Tested on giraffe:
- 9.3.5 cluster creation
- upgrade from 9.2.4

**Pull request merged**